### ISSUE TITLE
I can hide slick-disabled:before arrow, but the button steel keep there working. How can I hide and really disable next/prev action?  

### ISSUE THREAD

**Issue by pemasem** [kenwheeler/slick#3690]
> short description of the bug / issue, provide more detail below.

====================================================================

###### `[  paste your jsfiddle link here  ]`  

use this jsfiddle to reproduce your bug: 
http://jsfiddle.net/simeydotme/fmo50w7n/ 
we will likely close your issue without it.

====================================================================

#### Steps to reproduce the problem

1. ...  
2. ...  

====================================================================

#### What is the expected behaviour?

...  

====================================================================

#### What is observed behaviour?

...  

====================================================================

#### More Details

- Which browsers/versions does it happen on?
- Which jQuery/Slick version are you using?
- Did this work before?

**Comment by alimemonzx** [comment]
You have to change your settings with `arrows: false`

**Comment by pemasem** [comment]
I need the arrows, but I don't want it does nothing when disables.
It works even if I hide it

**Comment by alimemonzx** [comment]
@pemasem you are right. This is a bug even when arrows are false the slider is moving
